app/src/components/dashboard/CustomTooltip.tsx

```diff
@@ -0,0 +1,39 @@
+import { Card, Text, VStack } from "@chakra-ui/react";
+import { formatToUTCDayMonth } from "~/utils/dayjs";
+
+interface PayloadItem {
+  name: string;
+  color: string;
+  value: string;
+}
+
+function CustomTooltip({
+  active,
+  payload,
+  label,
+  beforeValue,
```

Comment:
Nit: `valuePrefix` might be a bit clearer.

Reply:
Nice, I like it! Changed.